lib/venue.rb

```diff
@@ -1,2 +1,36 @@
 class Venue
+  attr_reader :name,
+              :capacity,
+              :patrons
+
+  def initialize(name, capacity)
+    @name     = name
+    @capacity = capacity
+    @patrons  = []
+  end
+
+  def add_patron(patron)
+    @patrons << patron
+  end
+
+  def yell_at_patrons
+    upcased_patrons = patrons.map do |patron|
```

Comment:
Since you are using a `.map` here, you don't need to create a variable. The only time you would need to do that is if you have a nested iteration that you are working with.